.rubocop_todo.yml

```diff
@@ -181,15 +220,12 @@ Style/MutableConstant:
     - 'lib/qiita/markdown/embed/tweet.rb'
     - 'lib/qiita/markdown/filters/checkbox.rb'
     - 'lib/qiita/markdown/filters/code_block.rb'
-    - 'lib/qiita/markdown/filters/group_mention.rb'
-    - 'lib/qiita/markdown/filters/inline_code_color.rb'
-    - 'lib/qiita/markdown/filters/mention.rb'
     - 'lib/qiita/markdown/filters/simplify.rb'
```

Comment:
I verified the changes with the development environment and CI (GitHub Actions).
I would like to ask you to correct again some differences in the generated `.rubocop_todo.yml`.

```diff
diff --git a/.rubocop_todo.yml b/.rubocop_todo.yml
index 84d407e..3263685 100644
--- a/.rubocop_todo.yml
+++ b/.rubocop_todo.yml
@@ -210,7 +210,7 @@ Style/IfUnlessModifier:
     - 'lib/qiita/markdown/filters/truncate.rb'
     - 'lib/qiita/markdown/transformers/strip_invalid_node.rb'

-# Offense count: 14
+# Offense count: 17
 # This cop supports safe auto-correction (--auto-correct).
 # Configuration parameters: EnforcedStyle.
 # SupportedStyles: literals, strict
@@ -220,6 +220,9 @@ Style/MutableConstant:
     - 'lib/qiita/markdown/embed/tweet.rb'
     - 'lib/qiita/markdown/filters/checkbox.rb'
     - 'lib/qiita/markdown/filters/code_block.rb'
+    - 'lib/qiita/markdown/filters/group_mention.rb'
+    - 'lib/qiita/markdown/filters/inline_code_color.rb'
+    - 'lib/qiita/markdown/filters/mention.rb'
     - 'lib/qiita/markdown/filters/simplify.rb'
     - 'lib/qiita/markdown/filters/syntax_highlight.rb'
     - 'lib/qiita/markdown/version.rb'
```

Reply:
Thanks for the fix! 👍